app/assets/stylesheets/pages/announces_show.scss

```diff
@@ -67,4 +74,10 @@
   font-weight: 400;
   line-height: 26px;
 
+}
+
+.similar_announce {
+  font-size:30px;
+  text-align:center;
```

Comment:
Properties should be ordered font-size, line-height, text-align<br>Colon after property should be followed by one space